tideways/provision.sh

```diff
@@ -75,13 +75,24 @@ install_xhgui() {
     fi
 }
 
+enable_tideways_by_site() {
+    echo "Tideways-by-site started"
+
+    sudo apt install php-yaml -y
```

Comment:
We should get this in the core provisioner and the PHP utilities rather than here, lets keep as few `apt` install calls as possible and batch it all up together if we can

Reply:
So I will do a pr for that but I think that is better to check the pr is working before add that module